src/twisted/web/http.py

```diff
@@ -2296,6 +2300,17 @@ def timeOut(self, value):
         self._channel.timeOut = value
 
 
+    @property
+    def callLater(self):
```

Comment:
(Also thank goodness that `proxyForInterface` generates a class that inherits from `object`!)